Example/Backpack/View Controllers/HorizontalNavViewController.swift

```diff
@@ -27,15 +27,25 @@ class HorizontalNavViewController: UIViewController, BPKTappableLinkLabelDelegat
     var showIcons: Bool = false
     var wide: Bool = false
     var showExtraContent = false
+    var showNotificationDot = false
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
         if showIcons {
             horizontalNav.options = [
-                BPKHorizontalNavigationOption(name: "Flights", iconName: .flight, tag: 0),
-                BPKHorizontalNavigationOption(name: "Hotels", iconName: .hotels, tag: 1),
-                BPKHorizontalNavigationOption(name: "Car hire", iconName: .cars, tag: 2)
+                BPKHorizontalNavigationOption(name: "Flights",
+                                              iconName: .flight,
+                                              tag: 0,
+                                              showNotificationDot: showNotificationDot),
```

Comment:
Shouldn't only one of the tabs show the dot? Ideally the first one that's not selected i.e. `Hotels`?